model/postgres/postgres_server.rb

```diff
@@ -68,4 +68,9 @@ def hostname
   def connection_string
     URI::Generic.build2(scheme: "postgres", userinfo: "postgres:#{URI.encode_uri_component(superuser_password)}", host: hostname).to_s
   end
+
+  def inspect_values
+    hidden_columns = [:superuser_password, :root_cert_1, :root_cert_key_1, :root_cert_2, :root_cert_key_2, :server_cert, :server_cert_key]
```

Comment:
I'm brainstorming a similar concept for Clog. In the future, we might consider moving this variable to the class level.
https://github.com/ubicloud/ubicloud/pull/766

Reply:
Encrypted columns can be accessed with `column_encryption_metadata.keys` and we may hide them in the inspect output of all models.

Reply:
Let's do this. We used to have our own `Model` class (perhaps it should be named `CloverModel`) that all models inherited from, and then switched to `Sequel::Model` because we were doing nothing interesting in there, and then now may be good occasion to switch back again.

Reply:
Also let's improve the rendering of NetAddr columns, for the love of all that is holy (yes, we could also fork netaddr, and probably should at some point)